rero_ils/modules/acq_orders/jsonschemas/acq_orders/acq_order-v0.0.1.json

```diff
@@ -95,84 +95,15 @@
         ]
       }
     },
-    "description": {
-      "title": "Description",
-      "type": "string",
-      "minLength": 5
-    },
-    "order_status": {
-      "title": "Order status",
-      "type": "string",
-      "enum": [
-        "approved",
-        "canceled",
-        "ordered",
-        "requested",
-        "pending",
-        "received"
-      ],
-      "default": "pending",
-      "form": {
-        "type": "selectWithSort",
-        "options": [
-          {
-            "value": "approved",
-            "label": "approved"
-          },
-          {
-            "value": "canceled",
-            "label": "canceled"
-          },
-          {
-            "value": "ordered",
-            "label": "ordered"
-          },
-          {
-            "value": "requested",
-            "label": "requested"
-          },
-          {
-            "value": "pending",
-            "label": "pending"
-          },
-          {
-            "value": "received",
-            "label": "received"
-          }
-        ]
-      }
-    },
     "currency": {
       "title": "Order currency",
       "type": "string",
```

Comment:
Why do you replace the list of currencies by the free text of length 1 ???

Reply:
because we could use any currency than vendor define. We need to use a shared currency table between vendor and orders.  If we maintain two list in parallel, it will be bad very soon :-)